src/txmempool.cpp

```diff
@@ -620,12 +615,11 @@ static void CheckInputsAndUpdateCoins(const CTransaction& tx, CCoinsViewCache& m
 void CTxMemPool::check(const CCoinsViewCache *pcoins) const
 {
     LOCK(cs);
```

Comment:
Oops. You've added this back in your rebase.

Reply:
Thanks for catching that! Will fix 👍